Kitodo/src/test/resources/log4j2.xml

```diff
@@ -18,7 +18,7 @@
         </Console>
     </Appenders>
     <Loggers>
-        <Root level="debug">
+        <Root level="error">
```

Comment:
don't change the log level here

Reply:
Can’t fix. This is the setting which caused Travis to crash due to too long logs. Obviously the config file was not considered in the past and is considered now (how it shoud be!), don’t ask me why.

Reply:
Interesting that until now it was not crashing... It crashes only on this PR. Which stuff is logged here which overload the log? 

_Obviously the config file was not considered in the past and is considered now (how it shoud be!), don’t ask me why._

I don't get what you mean here. It is config file for test logging. Standard logging has own configuration. Even more you, as developer, have in config-local own config file which you can adjust to your needs.

Reply:
I'm going to merge it. I made a Jira Ticket to check log-level